src/Microsoft.DotNet.Build.Tasks/Targets/Build.Common.targets

```diff
@@ -8,6 +8,8 @@
     <AssemblyVersion Condition="'$(AssemblyVersion)'==''">999.999.999.999</AssemblyVersion>
     <CLSCompliant Condition="'$(CLSCompliant)'=='' and '$(IsTestProject)'=='true'">false</CLSCompliant>
     <CLSCompliant Condition="'$(CLSCompliant)'==''">true</CLSCompliant>
+
+    <ExcludeLocalizationImport Condition="'$(ExcludeLocalizationImport)'=='' And !Exists('$(MSBuildProjectDirectory)\MultilingualResources\$(MSBuildProjectName).de.xlf')">true</ExcludeLocalizationImport>
```

Comment:
Seems reasonable, be sure to call this out in the loc workflow doc.  You might also want to add a comment here explaining why we only check for the existence of de.  Other than that LGTM.